--- openS7online.c	Tue Jan  9 17:37:36 2007
+++ F:\lnd\libnodave-patches\openS7online.c	Fri May 17 10:06:24 2013
@@ -55,12 +55,45 @@
 
     en=SCP_get_errno();
     h=SCP_open(accessPoint);
+	/*if (h == -1)
+	{
     en=SCP_get_errno();
+		return en * -1;
+	}*/
     LOG3("handle: %d  error:%d\n", h, en);
 	SetSinecHWnd(h, handle);
     return h;
 };
     
+EXPORTSPEC int DECL2 getS7onlineErr()
+{
+    int en = SCP_get_errno();
+    return en;
+};
+   
+/*
+LRESULT CALLBACK HookProc(int nCode, WPARAM wParam, LPARAM lParam)
+{
+    //if (nCode >= 0) können wir uns sparen, da nCode eh niemals kleiner 0 sein wird!
+    /*MOUSEHOOKSTRUCT mhs = *(MOUSEHOOKSTRUCT*)lParam;
+    LONG lPoint = MAKELONG((WORD)mhs.pt.x, (WORD)mhs.pt.y);
+    HWND MyWin = FindWindow("TForm1", "Form1");
+    if (MyWin != NULL) //Das Fenster an das die Botschaft geschickt wird.
+        SendMessage(MyWin, WM_MOUSEMOVE, wParam, (LPARAM)lPoint);
+    return CallNextHookEx(0, nCode, wParam, lParam);*/
+/*}
+
+__declspec(dllexport) BOOL UninstallHook(HHOOK hook)
+{
+    return UnhookWindowsHookEx(hook); //Hook deinstallieren.
+}
+
+__declspec(dllexport) HHOOK InstallHook(HHOOK hook)
+{    //Hook installieren.
+    return SetWindowsHookEx(WM_SINEC, (HOOKPROC)HookProc, HInstance, 0);
+}*/
+
+
 EXPORTSPEC HANDLE DECL2 closeS7online(int h) {
     SCP_close(h);
 }
